release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -144,6 +144,10 @@ For example, in some scenarios, it is a priority to maximize CPU and memory reso
 
 For more information about this more flexible SR-IOV network deployment during NUMA-aware pod scheduling, see xref:../networking/hardware_networks/configuring-sriov-device.adoc#nw-sriov-exclude-topology-manager_configuring-sriov-device[Exclude the SR-IOV network topology for NUMA-aware scheduling].
 
+[id="ocp-4-14-networking-haproxy-update"]
+==== Update to HAProxy 2.6
+{product-title} updated to HAProxy 2.6.
```

Comment:
> With this release, {product-title} updated to HAProxy 2.6.

Added _With this release_ to match most others and I think you need the verb here.

Reply:
thanks! updated